stl/inc/yvals_core.h

```diff
@@ -934,6 +935,7 @@
 #define __cpp_lib_generic_unordered_lookup 201811L
 #define __cpp_lib_list_remove_return_type 201806L
 #define __cpp_lib_to_array 201907L
+#define __cpp_lib_bind_front 201910L

```

Comment:
This should be in sorted order by name. Additionally, its value should be `201907L` according to https://eel.is/c++draft/version.syn . (We set `_MSVC_STL_UPDATE` to the current month, but we use the Standard's values for its feature-test macros, which correspond to when those features were voted in. This can make a difference as the Standard values are occasionally increased by new papers being voted in.)

Reply:
Thanks for the help!